code/ui/lisibilite_ui_app/templates/scorecalculation.html

```diff
@@ -0,0 +1,102 @@
+{% load static %}
```

Comment:
Suggestion: Would be good to name it something like `userinput` or `textinput` - `scorecalculation` seems a bit off IMO

Reply:
Query: What does load static do?

Reply:
Will do the renaming as you suggested. It was userinput.html initially. I had changed it to the new name in the recent commit.

load static is is the general way of managing static files (like images, css, js etc) in django. It basically loads the path where these static files are stored. The path is defined in settings.py. You can read more here - https://docs.djangoproject.com/en/3.0/howto/static-files/.

Reply:
Pushed the changes related to the renaming of the file.